methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<p>I had trouble with the Path object on OSL, but maybe this change will work. I changed my OSL notebook to this:</p><pre class="ql-syntax" spellcheck="false">os.chdir(work_dir)

config_file_name = calval_location + '_Sen.txt'

config_file = os.path.join(mint_dir, config_file_name)

try:
    config_file_obj = mint_dir_obj/(calval_location+'_Sen.txt')
    config_file_obj.write_text(config_file_content)
except:
    with open(config_file,'w') as f:
        f.write(config_file_content)

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>